modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -387,37 +387,78 @@ For example, you cannot have one xref:indexes-vector-similarity-euclidean[Euclid
 == Known issues
 
 The vector search index is a beta feature.
-The below table lists the known issues in its current implementation. 
+The below table lists the known issues in its current implementation.
 
-[%header,cols="5d,d"]
+[%header,cols="5a,d"]
 |===
 | Known issues | Fixed in
 
-a| xref:clauses/listing-procedures.adoc[`SHOW PROCEDURES`] does not show the vector index procedures:
+| xref:clauses/listing-procedures.adoc[`SHOW PROCEDURES`] does not show the vector index procedures:
 
 * {link-procedures-reference}#procedure_db_create_setVectorProperty[`db.create.setVectorProperty`]
 * {link-procedures-reference}#procedure_db_index_vector_createNodeIndex[`db.index.vector.createNodeIndex`]
 * {link-procedures-reference}#procedure_db_index_vector_queryNodes[`db.index.vector.queryNodes`]
+
+[NOTE]
+====
+The procedures are still usable, just not visible.
+====
 | Neo4j 5.12
 
 | Passing `null` as an argument to some of the procedure parameters can generate a confusing exception.
-a| Neo4j 5.12
+| Neo4j 5.12
+
+| The creation of the vector index skipped the check to limit the dimensionality to `2048`.
```

Comment:
I guess we could also document 2048 as being the limit for what is *supported* and that it *might* be a hard limit in the future... this list is getting long 😄